mannequin fix for lizard people

FluffySensation opened the pull request:
Just a small sequence problem with the race default features being loaded after setting them and tail not being set to update.

This would lead to the mannequin just having a smooth tail and no spines the default setting for a lizard tail.

FluffySensation committed:
mannequin fix for lizard people

Just a small sequence problem with the race default features being loaded after setting them and tail not being set to update.

SpaceManiac commented on code/modules/client/preferences.dm:
```diff
@@ -1509,16 +1509,20 @@ GLOBAL_LIST_EMPTY(preferences_datums)
+	character.dna.features = features.Copy()
+	character.dna.real_name = character.real_name
+	
+	if(!features["tail_lizard"] == "None")
```
`!something` will only ever be `0` or `1`, never `"None"`

octareenroon91 replied:
Yeah you need another layer of parentheses here.

FluffySensation replied:
Well that's embarrassing, sorry about that should all be fix up in a moment

FluffySensation committed:
Mannequin Fix for Lizard People

Just a small sequence problem with the race default features being loaded after setting them and tail not being set to update.

This would lead to the mannequin just having a smooth tail and no spines the default setting for a lizard tail.

FluffySensation committed:
Merge remote-tracking branch 'upstream/master'
